doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -641,11 +641,6 @@ the General Response contents.
     print_itavs(itavs);
     sk_OSSL_CMP_ITAV_pop_free(itavs, OSSL_CMP_ITAV_free);
 
-=head1 SEE ALSO
-
-L<OSSL_CMP_exec_IR_ses(3)>, L<OSSL_CMP_exec_KUR_ses(3)>,
-L<OSSL_CMP_exec_GENM_ses(3)>
-
```

Comment:
Will you remember to tell those in the next CMP PR to addd this back?

Reply:
no, I put them back.  I will update #10256 to mark these as missing so the CMP PR that adds them will unmark them :)